### DISCUSSION TITLE
Class "Spatie\Backup\Helpers\Backup" not found

### DISCUSSION THREAD

**Discussion by jamesmalin** [spatie/laravel-backup#1703] [Q&A]
Receiving this error.
PHP 8.1.20
Laravel Framework 9.52.10
spatie/laravel-backup 8.2.0

**Comment by parallels999** [comment]
As I can see in your sample code, there is no sample code.
But look at https://github.com/spatie/laravel-backup/tree/main/src/Helpers, there is not any `Backup` class in there

Upgrade to spatie/laravel-backup 8.3.1
